main/modes/games/2048/mode_2048.c

```diff
@@ -232,30 +234,95 @@ static void t48MainLoop(int64_t elapsedUs)
             // Take accel input if cells aren't moving
             if (t48->tiltControls && !t48->cellsAnimating)
             {
-                int16_t ox, oy, oz;
-                if (ESP_OK == accelGetOrientVec(&ox, &oy, &oz))
+                // int16_t ox, oy, oz;
+                float q[4];
+                esp_err_t aq = accelGetQuaternion(q);
+                if (ESP_OK == aq)
                 {
-                    if (ABS(ox) > ABS(oy))
+                    float qRelativeRotation[4];
+
+                    // this takes about 36 flops.
+                    mathComputeQuaternionDeltaBetweenQuaternions(qRelativeRotation, t48->quatBase, q);
```

Comment:
I can't believe this has been around since e3858c5e4c671562f78395636f9f73ab37ce1a8b and I had no idea. I feel like such a doofus for missing it when trying to do tilt stuff myself